scipy/special/tests/test_orthogonal.py

```diff
@@ -93,6 +95,21 @@ def test_gegenbauer(self):
         assert_array_almost_equal(Ca5.c,array([4*sc.poch(a,5),0,-20*sc.poch(a,4),
                                                0,15*sc.poch(a,3),0])/15.0,11)
 
+    def test_n_zero_gh8888(self):
+        # gh-8888 reported that gegenbauer(0, 0) returns NaN polynomial
+        Ca0 = orth.gegenbauer(0, 0)
```

Comment:
Should we test this with other values of `alpha` too since `n=0` is the thing that determines that the polynomial is 1?